### ISSUE TITLE
Documentation issue in docker compose command for combnied importer setup

### ISSUE THREAD

**Issue by robot-army** [firefly-iii/firefly-iii#8073]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Hi, 

I think the documentation here: https://docs.firefly-iii.org/data-importer/installation/docker/ 

uses the wrong docker compose command. I think maybe it should be 

```docker compose -f docker-compose-importer.yml up -d --pull=always```

rather than ```docker compose -f docker-compose.yml up -d --pull=always```

### Debug information

n/a, documentation issue

### Expected behaviour

when running the docker compose command from the importer documentation, the importer and the main server should come up. When running the command as given, only the main firefly iii instance comes up, not the importer. Changing this one thing seems to bring up the importer.

**Comment by robot-army** [comment]
also, it says "In Firefly III at http://localhost/, browse to Profile > OAuth". It should read "Browse to options > profile > oauth", I think?

**Comment by JC5** [comment]
The instructions tell you to save the file as "docker-compose.yml" which is why the command is like that.

**Comment by robot-army** [comment]
Fair enough, thanks for the quick response! I think some clarity between the docs here: 
https://docs.firefly-iii.org/firefly-iii/installation/docker/

and here: 

https://docs.firefly-iii.org/data-importer/installation/docker/

might be good, or even a link from the first to the second for "if you want to install the data importer, go see here" or something. 

Anyways, all minor stuff, thanks again!

**Comment by JC5** [comment]
There is literally a link in the second line of the text ;)

https://docs.firefly-iii.org/firefly-iii/installation/docker/#
